accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -391,6 +391,13 @@ continuation.
 
 #### Late fields and variables
 
+A non-local variable declaration of the form `late final v;`, `late final T v;`,
```

Comment:
I think I'm in favor of not defining a setter here.

Reply:
What I'm proposing is that the forms with an initializer don't have a setter. But if the forms with _no_ initializer do not have a setter, how would they get initialized?

I would expect these forms to induce a setter (as specified in line 397), and that setter would perform the checks that enable dynamic initialization and prevent subsequent mutation.

Otherwise we'd need to come up with a substantially different specification of how to perform static checks and dynamic execution of assignments like `v = e`.

Reply:
Just noted that [this comment](https://github.com/dart-lang/language/issues/523#issuecomment-553192653) seems to express support for (1) `late final` with initializer does not induce a setter, and (2) `late final` without initializer does induce a setter, which is what this PR specifies. So maybe we agree already.

Reply:
Agreed.